apps/odyssey-stats/src/styles/wp-admin.scss

```diff
@@ -6,6 +6,37 @@
 	}
 }
 
+// Override notice styles
+#wpbody-content > .notice {
```

Comment:
I'm using `#wpbody-content >` here so this gets applied only to the banners injected by WP or Jetpack and doesn't affect the use of the Notice component inside Odyssey Stats.